_posts/2019-10-26-dividing-human-pie-ai.md

```diff
@@ -0,0 +1,59 @@
+
+Virtually every organisation that we meet is trying to figure out how to use machine learning and artificial intelligence. So much hype and expectation has evolved around these exciting technologies that it can be hard to cut through the noise and understand what they are, how they really work, and when they should be used.
+
+To help organisations to address these questions Infinity Works recently started to run machine learning immersion days in association with [Amazon Web Services](https://aws.amazon.com/). 
+
+We invite developers, engineering managers and business leaders who are interested in using AI to a day of talks and hands-on workshops. We provide them with a holistic view of the software delivery lifecycle for machine learning, and how to build and deploy machine learning algorithms in the AWS cloud.
+
+AWS provides some great example exercises that demonstrate the capabilities of [Amazon SageMaker](https://aws.amazon.com/sagemaker/), [Amazon Lex](https://aws.amazon.com/lex/), [Amazon Personalize](https://aws.amazon.com/personalize/) and [Amazon Forecast](https://aws.amazon.com/forecast/). In the afternoon session participants worked through a choice of these exercises at their own pace while a colleague and I offered help and explanation if they got stuck.
+
+Feedback from a recent event indicated that participants would've liked more guidance in the hands-on session. We're always keen to try new things, figure out what works and what doesn't, fix things quickly and continuously improve. However, opinions were split on how to improve the experience for participants.
+
+* Some participants would have preferred more explanation of how the AWS cloud works, and that they'd have appreciated working more slowly when configuring the infrastructure.
+
+* Some developers felt that we'd discussed the data science aspects too briefly, so they didn't have time to pick up any new knowledge in that area.
+
+* Another respondent said that the machine learning content could have been more challenging, because we'd focused on the basics.
+
+## How to divide a human pie?
+Dividing the participants into two groups would give the participants more contact time with the helpers. The helpers would also have some data to help them to tailor their guidance to address their group's knowledge gaps. Asking for help always feels easier in smaller groups, too. 
+
+But given the participants' diversity of experience, how should we divide the groups to give everyone a better experience?
+
+There were so many possible ways to divide our human pie that there was no clear answer. I needed some data!
+
+During a morning break I asked all of the participants to rate themselves in terms of their machine learning knowledge, where on the developer - data scientist spectrum they identify themselves, and how much cloud experience they had, using a Google Form.
```

Comment:
@mcmont, could you give an example (maybe a screenshot) of the Google form?

Reply:
Yep! I've added a screenshot. Another image helps to break up the text too, which is nice.